plugins/node/instrumentation-fs/src/instrumentation.ts

```diff
@@ -415,4 +407,23 @@ export default class FsInstrumentation extends InstrumentationBase<FS> {
       }
     }
   }
+
+  protected _shouldTrace(): boolean {
+    const activeContext = api.context.active();
```

Comment:
suggestion: you can move the `activeContext = api.context.active()` to be a function argument. This has 2 potential benefits:
1. It communicates that the function depends on an external value `context` instead of reading it from the global scope as side effect. This makes the function dependencies explicit instead of implicit. If the value is hoisted to the function signature, it depends only on `this` and function arguments which might be simpler to maintain and understand.
2. It is more consistent with other places in the project where the context is passed as a function argument.

This is completely optional and not blocker